gammapy/image/radial_profile.py

```diff
@@ -0,0 +1,87 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Image utility functions"""
+import logging
+import numpy as np
+from astropy.coordinates import Angle
+from astropy.wcs.utils import skycoord_to_pixel
+from astropy.table import QTable
+from scipy.ndimage.measurements import labeled_comprehension
```

Comment:
Put this as a delayed import into the function where you use it (on the first line these).
This is our coding standard for optional dependencies like scipy.

Travis-ci tests will fail otherwise.